pkg/webhook/podprobemarker/validating/probe_create_update_handler.go

```diff
@@ -245,6 +245,25 @@ func validateTCPSocketAction(tcp *corev1.TCPSocketAction, fldPath *field.Path) f
 	return ValidatePortNumOrName(tcp.Port, fldPath.Child("port"))
 }
 
+var supportedHTTPSchemes = sets.New(corev1.URISchemeHTTP, corev1.URISchemeHTTPS)
+
+func validateHTTPGetAction(http *corev1.HTTPGetAction, fldPath *field.Path) field.ErrorList {
+	allErrors := field.ErrorList{}
+	if len(http.Path) == 0 {
+		allErrors = append(allErrors, field.Required(fldPath.Child("path"), ""))
```

Comment:
shall we validate the format of path e.g. url.Parse

Reply:
url.Parse support parse relative path(a path, without a host), and that is exactly what formatURL in probe.go do. I think we can  add more strict checking in Kruise, since we don't have some legacy cases to support.

Reply:
Actually, okay, I will finish it.